### PR TITLE
Feature/#3

### LINKED ISSUE #3: add list app

### PR DESCRIPTION
**Pull request by StepanTchynetskyi**

### CODE REVIEW

**Review by SLDem — APPROVED** [review]

File: list_project/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
Comment (lhalam) [inline list_project/admin.py]: delete this file

File: list_project/urls.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from django.urls import path
+from . import views
+urlpatterns = [
+    path('',views.index, name='index'),
+]
```
Comment (lhalam) [inline list_project/urls.py]: add endline

File: list_project/urls.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+from django.urls import path
+from . import views
+urlpatterns = [
+    path('',views.index, name='index'),
```
Comment (lhalam) [inline list_project/urls.py]: remove this line

File: list_project/views.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+# Create your views here.
+def index(request):
```
Comment (lhalam) [inline list_project/views.py]: add endline

File: project_config/urls.py [hunk 5]
```diff
@@ -13,8 +13,10 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.urls import path
+from django.urls import path,include
 
-urlpatterns = [
 
+urlpatterns = [
+    path('', include('list_project.urls')),
```
Comment (lhalam) [inline project_config/urls.py]: update to `path('list_project', include('list_project.urls'))`,

File: list_project/urls.py (hunk 3 above)
Comment (StepanTchynetskyi) [reply list_project/urls.py]: done

File: list_project/urls.py (hunk 2 above)
Comment (StepanTchynetskyi) [reply list_project/urls.py]: done

File: list_project/admin.py (hunk 1 above)
Comment (StepanTchynetskyi) [reply list_project/admin.py]: done

File: list_project/views.py (hunk 4 above)
Comment (StepanTchynetskyi) [reply list_project/views.py]: done

File: project_config/urls.py (hunk 5 above)
Comment (StepanTchynetskyi) [reply project_config/urls.py]: done

**Review by lhalam — APPROVED** [review]

**Pull request merged**